src/translator_testing_model/datamodel/pydanticmodel.py

```diff
@@ -53,6 +53,10 @@ class TestObjectiveEnum(str, Enum):
     BenchmarkTest = "BenchmarkTest"
     # Quantitative test
     QuantitativeTest = "QuantitativeTest"
+    # One Hop Tests of knowledge graph navigation
+    OneHopTests = "OneHopTests"
```

Comment:
Should this be plural?

Reply:
I guess not, for consistency. I'll change that. Done